src/Microsoft.AspNet.Mvc.TagHelpers/LinkTagHelper.cs

```diff
@@ -98,6 +108,12 @@ private enum Mode
         public string FallbackHref { get; set; }
 
         /// <summary>
+        /// Value indicating if file version should be appended to the href urls.
+        /// </summary>
+        [HtmlAttributeName(FileVersionAttributeName)]
+        public string FileVersion { get; set; }
```

Comment:
Thought about this some more and decided it should be `false` by default actually.

Also, this has to be a `bool`